src/context/filterContext/filterReducer.jsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+
```

Comment:
You can create a variable 
```
export const initialState = {
    highToLow: false,
    lowToHigh: false,
    discount: {
      "10_OR_BELOW": false,
      "10_OR_ABOVE": false,
      "30_OR_ABOVE": false,
    },
    category: { racquets: false, shoes: false, apparel: false, bags: false },
    ratings: 0,
    priceRange: 2000,
  }

```
Use it for reset filter also

Reply:
done